src/datastore/common.ts

```diff
@@ -1379,6 +1379,7 @@ export interface SmartContractEventInsertValues {
   contract_identifier: string;
   topic: string;
   value: PgBytea;
+  value_json: string;
```

Comment:
It would be best if we set this as `string | null` IMO to avoid storing empty strings